rskj-core/src/main/java/org/ethereum/vm/program/Program.java

```diff
@@ -1351,7 +1349,7 @@ public int verifyBeginSub(DataWord nextPC) {
         return ret;
     }
 
-    public void callToPrecompiledAddress(MessageCall msg, PrecompiledContract contract, ActivationConfig.ForBlock activations) {
+    public void callToPrecompiledAddress(MessageCall msg, PrecompiledContract contract) {
```

Comment:
No PrecompiledContractException throws? Why?

Reply:
We wanted to keep consistency with the behaviour of "callToAddress()", which handles all the errors internally, it doesn't bubble up the error but push a Zero into the stack.